includes/class-wc-stripe-checkout-session-lifecycle.php

```
@@ -0,0 +1,244 @@
+	 */
+	private function get_classic_fragment_html( array $data ): string {
+		$encoded_data = wp_json_encode( $data );
+		if ( false === $encoded_data ) {
+			$encoded_data = '{}';
+		}
+
+		return sprintf(
+			'<div id="wc-stripe-checkout-session-data" data-checkout-session="%s" hidden></div>',
+			esc_attr( $encoded_data )
+		);
+	}
```

PHPMD (2.15.0)

[error] The method get_classic_fragment_html is not named in camelCase. (undefined)

(CamelCaseMethodName)